scarlet2/nn.py

```diff
@@ -96,16 +96,16 @@ def calc_grad(x, model, model_size=32):
     -------
     score_func : array of the score function
     """
-    # perform padding if needed
+    t = 0.02 # temperature: here we set a fixed value which should only be changed if the user is familiar with the model
```

Comment:
I dislike this implicit setting. The users should indeed by able to set this themselves, but than can simply be achieved by creating a `__call__` function in the model specification that knows of the temperature. E.g. 
```
class ModelWithTemp:
  def __init__(self, model, temp):
    ...
  def __call__(self, x):
    return self.model(x, t=self.temp)
```